### DISCUSSION TITLE
copy and paste not working for written lessons on Cyfrin Updraft

### DISCUSSION THREAD

**Discussion by pythondefichain** [Cyfrin/foundry-full-course-cu#2856] [Q&A]
Can someone explain why copy and paste does not work on written lessons on the CyfrinUpdraft learning website?

**Comment by EngrPips** [comment]
Hello @pythondefichain, I am unsure what you mean. Do you mean you can't copy stuff from Cyfrin Updraft's written lesson?

**Comment by LordAdhaar** [comment]
Can you link the webpage it is not working for. I tried right now, seems to be working fine for me.
